compiler/modules/nback/build.bal

```diff
@@ -1343,6 +1343,19 @@ function buildEqualFloat(llvm:Builder builder, Scaffold scaffold, boolean exact,
     return buildStoreBoolean(builder, scaffold, b, reg);
 }
 
+function generateStringEquality(llvm:Builder builder, Scaffold scaffold, CmpEqOp op, bir:EqualityInsn insn, llvm:Value lhsValue, llvm:Value rhsValue, bir:Register result) {
+    foreach var operand in insn.operands {
+        if !(operand is string) {
+            continue;
+        }
+        string str = <string>operand;
+        if isSmallString(str.length(), str.toBytes(), str.toBytes().length()) {
```

Comment:
Don't call str.toBytes twice: each call does the conversion to UTF-8.

Reply:
Fixed